packages/typespec-ts/package.json

```
@@ -30,6 +30,7 @@
     "generate-tsp-only:non-branded": "npm run generate-tsp-only:non-branded-rlc && npm run generate-tsp-only:non-branded-modular",
     "generate-tsp-only:non-branded-rlc": "node --loader ts-node/esm ./test/commands/gen-cadl-ranch.ts --tag=non-branded-rlc",
     "generate-tsp-only:non-branded-modular": "node --loader ts-node/esm ./test/commands/gen-cadl-ranch.ts --tag=non-branded-modular",
+    "regen-test-baselines": "npm run generate-tsp-only",
```

```suggestion
    "regen-test-baselines": "npm run generate-tsp-only && npm run generate-tsp-only:non-branded",
```
or we could change the generate-tsp-only script to include the non-branded ?